CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Added
 
 ### Changed
+- [\#329](https://github.com/Manta-Network/manta-signer/pull/329) Fix bad sync after reimport or password reset
```

Comment:
I think this changelog is messed up, the current version is 1.1.1, this should be "Unreleased." Should also probably go back and add the changes for 1.1.1, either in this PR or the release.

Reply:
Yes a merge seems to have messed it up